src/cljs/rems/application.cljs

```diff
@@ -784,7 +784,10 @@
       (when comment
         [:div.form-control.event-comment comment])
       (when-some [attachments (seq @(rf/subscribe [::get-attachments-by-event-id event-id]))]
-        [fields/attachment-row attachments {:compact true}])]]))
+        (into [:<>]
+              (->> attachments
+                   (partition-all 3)
+                   (map fields/attachment-row))))]]))
```

Comment:
If `attachment-row` is a component, it should be wrapped in a vector and not called as a function?

Should this just be a regular `for`?

Reply:
i don't think there is any reason why this would not have worked, but refactored regardless. both `map` and `for` return sequence, which is realized by `into`?